assets/css/styles.scss

```diff
@@ -12,6 +12,37 @@ body {
   font-style: normal;
 }
 
+.blog {
+  margin: 50px auto;
+  max-width: 800px;
+  line-height: 2.5rem;
+}
+
+.blog p {
```

Comment:
In scss, you write it in cascading style with indentation. So you won't be defining a `.blog p` but rather `p` inside `.blog`. Please see rest of the code file to understand better.